src/test/java/org/folio/rest/impl/APITests.java

```diff
@@ -0,0 +1,113 @@
+package org.folio.rest.impl;
+
+import static java.lang.String.format;
+
+import java.lang.invoke.MethodHandles;
+import java.util.Base64;
+import java.util.Collections;
+import java.util.UUID;
+
+import org.folio.rest.RestVerticle;
+import org.folio.rest.client.TenantClient;
+import org.folio.rest.jaxrs.model.Parameter;
+import org.folio.rest.jaxrs.model.TenantAttributes;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.PomReader;
+import org.folio.rest.tools.utils.NetworkUtils;
+import org.folio.rest.utils.OkapiClient;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+
+import com.github.tomakehurst.wiremock.common.ConsoleNotifier;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+
+public class APITests {
+  protected static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  private static final String MODULE_NAME_TEMPLATE = "%s-%s";
+  private static final String HTTP_PORT_PROPERTY = "http.port";
+
+  protected static final int OKAPI_PORT = NetworkUtils.nextFreePort();
+  protected static final String OKAPI_URL = "http://localhost:" + OKAPI_PORT;
+  protected static final String OKAPI_TENANT = "test_tenant";
+  protected static final String OKAPI_TOKEN = buildToken();
+  protected static final String OKAPI_HEADER_URL = "x-okapi-url";
+
+  protected static Vertx vertx;
+  protected static OkapiClient okapiClient;
+  protected static TenantClient tenantClient;
+
+  @Rule
+  public WireMockRule wireMock = new WireMockRule(
```

Comment:
Move wireMock to the class where it's being used

Reply:
done